css/properties.json

```diff
@@ -1282,7 +1282,7 @@
     "status": "nonstandard"
   },
   "-webkit-mask-origin": {
-    "syntax": "[ padding | border | content ] [, [ border | padding | content ] ]*",
+    "syntax": "[ <box> | fill-box | stroke-box | view-box | no-clip ]#",
```

Comment:
What is your source of the syntax?
`-webkit-mask-origin` is not a standard property, and I can only find something like http://css-infos.net/property/-webkit-mask-origin